tests/ldevnet.go

```diff
@@ -39,7 +39,7 @@ func LaunchDevnetDocker(t TestingTWithCleanup, numMiners int, ipfsMaddr string,
 	}
 
 	repository := "textile/lotus-devnet"
-	tag := "ntwk-calibration-8.8.0"
+	tag := "ntwk-calibration-8.13.1.f0942"
```

Comment:
also just flagging this one to check

Reply:
Oh, yeah, they're the same. Just that one tag name got a bit longer/shorter than the other.
One is the `textile/local-devnet` and the other `textile/lotus`.

The prefix of both is the same. I'm understanding correctly your point? (just want to double-check that was your point)

Reply:
Oh yea they are the same, didn't notice. All good!